Describe payment card support (#36)
MewsSystems/gitbook-distributor-guide: distributor-api-v1/general-guidelines.md

To do that some Distributor API endpoints support or require [CreditCardData](operations.md#creditcarddata) (represents payment card data with name `CreditCardData` for legacy and compatibility reasons) in the request. You can visit [Operations](operations.md) to see which endpoints they are.

To send the correct [CreditCardData](operations.md#creditcarddata) you need (check the [docs](operations.md#creditcarddata) to see which are required):
* Card holder name
* Card number
* CVV
* Expiration date

This is the relation between the human readable names and the field names which Distributor API uses:

| Human readable      | Distributor API       |
| :------------------ | :-------------------- |
| Card holder name    | HolderName            |
| Card number         | PaymentGatewayData    |
| Expiration date     | Expiration            |
| CVV                 | PaymentGatewayData    |

> `Expiration` and `HolderName` need no special handling. You can just get them from the user in any way you want and send them to the Distributor API as is.

Reviewer: ```suggestion
`Expiration` and `HolderName` are not sensitive data in terms of PCI DSS and plain text can be used. The expiration does need to follow a format that is described below.
```

Author: used with some edits

> `PaymentGatewayData`, which is an encoded string representing card number and CVV, needs more setup. 

Reviewer: ```suggestion
`PaymentGatewayData`, represents encoded string via payment card storage provider (LINK TO THE https://github.com/MewsSystems/gitbook-distributor-guide/blob/master/distributor-api-v1/payment-gateway-data.md). Those data are sensitive in terms of PCI DSS and therefore the implementation can not handle it in a plain text. 
```

Author: used with some edits

> Because of security reasons you can't just let user enter payment card data such as CVV and card number and send them to Distributor API.

Reviewer: I would remove it from here and describe it as mentioned in the previous comment

Author: will remove

> `PaymentGatewayData`, represents encoded string via payment card storage provider (LINK TO THE https://github.com/MewsSystems/gitbook-distributor-guide/blob/master/distributor-api-v1/payment-gateway-data.md). Those data are sensitive in terms of PCI DSS and therefore the implementation can not handle it in a plain text. 

Reviewer: ```suggestion
`PaymentGatewayData`, represents encoded string via [payment card storage provider](https://github.com/MewsSystems/gitbook-distributor-guide/blob/master/distributor-api-v1/payment-gateway-data.md). Those data are sensitive in terms of PCI DSS and therefore the implementation can not handle it in a plain text. 
```

Author: will fix link

> Instead you need to:
> * Collect and store payment card data securely with PCI Proxy.
> * Receive `transactionId` from PCI Proxy.
> * Use `transactionId` and other collected payment card data mentioned earlier to create and send [CreditCardData](operations.md#creditcarddata) to Distributor API.
> * Mews can then use [CreditCardData](operations.md#creditcarddata). For example for charging the user.

Reviewer: ```suggestion
Using PCIProxy is preffered way how to obtain the card sensitive data (https://docs.pci-proxy.com/collect-and-store-cards/capture-iframes). Following the steps in the documentation should guide you through the process of obtaining the `PaymentGatewayData
```

Author: used the idea, tried to fit it with the rest of the text
